mne_bids/io.py

```diff
@@ -83,3 +92,75 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
         raise ValueError('Raw file name extension must be one of {}\n'
                          'Got {}'.format(ALLOWED_EXTENSIONS, ext))
     return raw
+
+
+def read_raw_bids(bids_fname, bids_root, return_events=True,
+                  verbose=True):
+    """Read BIDS compatible data.
+
+    Parameters
+    ----------
+    bids_fname : str
+        Full name of the data file
+    bids_root : str
+        Path to root of the BIDS folder
+    return_events: bool
+        Weather to return events or not. Default is True.
+    verbose : bool
+        The verbosity level
+    """
```

Comment:
(Sorry this only just went through now... I though when you clicked submit it would actually submit...) Probably didn't need it to be a review either... stupid github I don't know...

Reply:
indeed. See: https://github.com/mne-tools/mne-bids/pull/181